relaxed.install

```diff
@@ -19,8 +19,19 @@ function relaxed_install() {
       'permissions' => [
         'perform push replication',
         'administer users',
-        'administer workspaces'
+        'administer workspaces',
+        'bypass node access',
       ],
     ])->save();
   }
 }
+
+/**
+ * Load new rest configuration for RESTful Web Services module.
+ */
+function relaxed_update_8301() {
```

Comment:
I think we should just remove this anyway? People upgrading would have already run this and REST config does not exist now with this work.